### PR TITLE
Removing deprecated warning

### LINKED ISSUE #202: CakePHP 3.7 Deprecation Warning 3.13.1
Just upgraded to CakePHP 3.7.1 and getting the following deprecated warning

```
Deprecated (16384): Checking a single plugin with Plugin::loaded() is deprecated. Use Plugin::isLoaded() instead. - /vendor/dereuromark/cakephp-queue/src/Model/Table/QueuedJobsTable.php, line: 87
```

### PR DESCRIPTION
**Pull request by challgren**
Fixes #202

### CODE REVIEW

**Comment by challgren** [comment]
Standby found some more instances

**Comment by challgren** [comment]
Ok should be good to go now

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	],
 	"require": {
 		"php": ">=5.6",
-		"cakephp/cakephp": "^3.6"
+		"cakephp/cakephp": "^3.7"
```
Comment (dereuromark) [inline composer.json]: If we need this, we also need to adjust the readme.

Comment (challgren) [reply composer.json]: `Plugin::isLoaded(string)` is a 3.7.0 only feature I believe. I can't really find when `isLoaded` was introduced

Comment (challgren) [reply composer.json]: Or maybe we can submit a backport of `Plugin::isLoaded(string)` to cakephp/cakephp.

Comment (challgren) [reply composer.json]: Would you be opposed as to me implementing a method along `isLoaded()` but in the package, that would do a check based on the CAKE_VERSION and call the appropriate method based on the version?

Comment (dereuromark) [reply composer.json]: i am just surprised to see tests green after updating the template and before raising the min version.

Comment (challgren) [reply composer.json]: I looked at the build and it appears composer downgraded to 3.7.0 when --prefer-lowest was provided https://travis-ci.org/dereuromark/cakephp-queue/jobs/470218616

**Pull request merged**